R/clean_clover_hosts.R

```diff
@@ -11,8 +11,14 @@ clean_clover_hosts <- function(virion_ictv_ratified){
     dplyr::mutate(HostFlagID = tidyr::replace_na(HostFlagID, FALSE)) 
   
   virion_ictv_ratified %<>%
-    mutate_cond(stringr::str_detect(HostOriginal, " cf\\."), HostFlagID = TRUE) 
```

Comment:
@cjcarlson I think the spirit of this function was the same as case when. For some unknown reason it started throwing an error so I changed it to case_when and now it seems to be working :)

Reply:
The deleted line was flagging hosts with a potential open nomenclature issue, so as long as everything that _Something cf. something else_ gets flagged somewhere else in the code, removing it should be fine?